ui/page/dexclient/create_wallet_modal.go

```diff
@@ -94,6 +94,8 @@ func (md *createWalletModal) OnResume() {
 }
 
 func (md *createWalletModal) Handle() {
+	md.modal.SetDisabled(md.isSending)
```

Comment:
Where is the value of `md.isSending` modified? That's where `md.modal.SetDisabled` should be called too.